cmd/dockerfile/digests.go

```diff
@@ -0,0 +1,72 @@
+package dockerfile
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/buildsafedev/bsf/cmd/styles"
+	"github.com/google/go-containerregistry/pkg/crane"
+	"github.com/spf13/cobra"
+)
+
+var DGCmd = &cobra.Command{
+	Use:     "dockerfile digests",
```

Comment:
`digests` should be a subcommand of `dockerfile`. 
`dockerfile` should still have `df` as shortform.

Reply:
Something like this 
```go
if args[0] == "digests" { // All my stuff }
```

Reply:
No, look at `bsf att`. We have subcommands defined there as well.

Reply:
gotcha👍